test/t8_test_cmesh_copy.c

```diff
@@ -37,40 +47,249 @@ test_cmesh_committed (t8_cmesh_t cmesh)
   SC_CHECK_ABORT (retval == 1, "Cmesh face consistency failed.");
 }
 
+
+/*
+ *The functions t8_get_*_cmesh_testcases return the number of 
+ *testcases for a given cmesh type.
+ */
```

Comment:
Nice comment.

Reply:
I think all these getter functions should be moved into t8_cmesh.c.  Furthermore the names of the these functions are a little missleading. I propose to call them something like t8_get_number_of_testname_testcases.